actors/miner/src/lib.rs

```diff
@@ -394,38 +399,41 @@ impl Actor {
     {
         let current_epoch = rt.curr_epoch();
 
-        if params.proofs.len() != 1 {
-            return Err(actor_error!(
-                ErrIllegalArgument,
-                "expected exactly one proof, got {}",
-                params.proofs.len()
-            ));
-        }
+        {
+            let policy = rt.policy();
```

Comment:
nit: move to directly precede the code block using it

Reply:
same problem here.